pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -164,8 +164,8 @@ message ServerConfiguration {
   // inside directories that have access controls set up properly.
   repeated string listen_paths = 2;
 
-  // TLS configuration. TLS is not enabled when left unset.
-  buildbarn.configuration.tls.ServerConfiguration tls = 3;
+  // was TLS, moved tl TLS option in transport_credentials.
+  reserved 3;
```

Comment:
No need to mark 3 as reserved. Just let `tls` below use field number 3 again. With Protobuf it's safe to upgrade a single field to a `oneof`, provided you preserve the field number.